friktion/friktion/config.py

```diff
@@ -2,7 +2,6 @@
 from decimal import Decimal
 from typing import Any
 
-import solders.keypair
```

Comment:
It is used at line 140
` wallet = Wallet(Keypair(solders.keypair.Keypair.from_base58_string(private_key)))`

should not be removed